examples/lighting-app/esp32/README.md

```diff
@@ -154,10 +154,11 @@ After commissioning is successful, press Enter in requestor device console and
 type below query.
 
 ```
->matter ota query 1 12345 0
+>matter ota query 1 12346 0
 ```
 
 ## Apply update
 
-Once transfer is complete, reboot the device manually to boot from upgraded OTA
-image.
+Once the transfer is complete OTA requestor sends ApplyUpdateRequest command to
```

Comment:
nit
```suggestion
Once the transfer is complete, OTA requestor sends ApplyUpdateRequest command to
```

Reply:
I believe this comma is still missing?

Reply:
I think I added the suggestions to batch and forgot to submit. Submitted now.